types.go

```diff
@@ -215,6 +215,27 @@ type (
 	}
 )
 
+type (
+	PackageBuildRequest struct {
+		SrcPkgFilename string `json:"srcPkgFilename"`
+		// Command for builder to run with.
+		// A build command consists of commands, parameters and environment variables.
+		// Now, only two environment variables is support:
+		// 1. SRC_PKG: path to source package directory
+		// 2. DEPLOY_PKG: path to deployment package directory
+		// The environment variables will be replaced with correspond package directory path.
```

Comment:
I don't think you need to explain this, it's just normal shell command variable substitution. I suggest removing the examples too, it makes the reader think there's something more complex going on.